thundra/integrations/botocore.py

```diff
@@ -243,14 +266,14 @@ def getRequestType(self, string):
         return Constants.AWS_SERVICE_REQUEST
 
     def inject_span_info(self, scope, wrapped, instance, args, kwargs, response,
-                 exception):
+                         exception):
 
         operation_name, request_data = args
         self.deliveryStreamName = request_data['DeliveryStreamName']
 
         scope.span.domain_name = Constants.DomainNames['STREAM']
         scope.span.class_name = Constants.ClassNames['FIREHOSE']
-        scope.span.operation_name = self.getRequestType(operation_name)
+        scope.span.operation_name = self.deliveryStreamName
```

Comment:
what if there is no `deliveryStreamName` in the request? For example, when the request is for list streams